app/src/main/java/com/myapp/travelize/authentication/LoginFragment.kt

```diff
@@ -93,16 +112,16 @@ class LoginFragment : Fragment() {
 
     }
 
-    private fun validate(email: String, password: String) {
+    private fun validate(email: String, password: String) : Boolean {
         if (TextUtils.isEmpty(email) || TextUtils.isEmpty(password)) {
             Toast.makeText(activity, "Some Field is Empty!", Toast.LENGTH_SHORT).show()
-            return
+            return false
```

Comment:
since you have disabled the sign in button on an empty email/password, this if statement is effectively redundant so can you remove it?